gallery/src/GalleryNav/GalleryNav.scss

```diff
@@ -0,0 +1,13 @@
+.gallery-nav-link {
+  color: var(--nx-color-text-dark);
+
+  .fa-sass {
+    // Optical horizontal alignment adjustment
+    margin-left: calc(var(--nx-spacing-2x) + 0.08em);
+  }
+}
+
+.gallery-nav-icon {
+  margin: 0 var(--nx-spacing-2x);
+  vertical-align: middle;
```

Comment:
How well this works depends a lot on where the shapes are within the SVG, but as it stands here I feel like it's a tad bit low.  FA uses `-0.125em` as their vertical alignment, so eyeballed numbers are a valid approach.  For this icon, -0.3em` looks about right to me

Reply:
> How well this works depends a lot on where the shapes are within the SVG, but as it stands here I feel like it's a tad bit low. FA uses `-0.125em` as their vertical alignment, so eyeballed numbers are a valid approach. For this icon, `-0.3em` looks about right to me

Agreed: https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/17d72c3b90b470ff8630aa7aa99ffda4c5af6b8c